core/libp2p/muxer/yamux/yamux_frame.cpp

```diff
@@ -0,0 +1,134 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "libp2p/muxer/yamux/yamux_frame.hpp"
+
+#include <arpa/inet.h>
+
+namespace libp2p::muxer {
+  kagome::common::Buffer YamuxFrame::frameBytes(
+      uint8_t version, FrameType type, Flag flag, uint32_t stream_id,
+      uint32_t length, const kagome::common::Buffer &data) {
+    return kagome::common::Buffer{}
```

Comment:
I strongly suggest using some sort of stream.
I think it would be better to write `NetworkOrderEncoder`, which will push numbers/whatever to a stream rather than using buffer

Reply:
Created a ToDo for it